src/docs/product/error-monitoring/index.mdx

```diff
@@ -1,7 +1,30 @@
 ---
-title: Error Monitoring
+title: Issues
 sidebar_order: 1
-description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in sentry.io."
+description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in [sentry.io](https://sentry.io)."
 ---
 
+The Issues page of [sentry.io](https://sentry.io) is where you can view, triage, and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.
+
+The Issues page is organized into tabs, each corresponding to a filtered list of issues:
+
+- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the For Review tab
+- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed
+- Ignored (`is:ignored`): All ignored issues
+- Saved Searches: Select from a set of pre-built and custom saved searches. The name of the tab changes based on your selection.
+
+TO DO identify what link for last bullet above should be and add it
+TO DO add screenshot here of issue stream
+
+From this page, you can begin to an issue. For information about triaging issues and their different states, see [Issue States and Triage](/product/error-monitoring/states-triage/).
+
+For each issue, the page displays several pieces of information including the following:
+
+- Error type and message
+- Brief description of the issue
+- Associated project
+- Last time the issue was seen
+- First time the issue was seen
```

Comment:
Perhaps this might be better captured as follows to allow for changes in the UI or expansion?
```suggestion
- Issue timing, such as last and first time seen
```